tests/data/make_central_repo.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash -e
+# Create a central repo that contains calibs, refcats, templates for use in
+# tests, making all the files be empty and adjusting the sqlite registry to
+# match.
+
+butler create central_repo
+butler import --export-file export.yaml --transfer copy central_repo/ $AP_VERIFY_CI_HITS2015_DIR/preloaded/
```

Comment:
Note that the `export.yaml` will bit-rot as we make changes to the `ap_verify` dataset. Might it be better to commit the export command instead of the file?

Reply:
I also had to tweak the export file to include the CHAINED collections that we want. I'll add a note here about that, and the command I used to run it.

Reply:
Yikes. And I guess there's no time to test a file that includes the chains from the beginning...